src/Shopsys/ShopBundle/Resources/translations/messages.de.po

```diff
@@ -2,884 +2,899 @@ msgid ""
-msgstr "(de) Tax number"
+msgstr "Steuernummer"
 
 msgid "Technical parameters"
-msgstr "(de) Technical parameters"
+msgstr "Teschnische Parameter"
 
 msgid "Telephone"
-msgstr "(de) Telephone"
+msgstr "Telefon"
 
 msgid "Thank you, your message has been sent."
-msgstr "(de) Thank you, your message has been sent."
+msgstr "Vielen Dank! Ihre Nachricht wurde gesendet."
 
 msgid "The link to change your password expired."
-msgstr "(de) The link to change your password expired."
+msgstr "Der Link zum Ändern Ihres Passworts ist abgelaufen."
 
 msgid "The price of payment {{ paymentName }} changed during ordering process. Check your order, please."
-msgstr "(de) The price of payment {{ paymentName }} changed during ordering process. Check your order, please."
+msgstr "Der Preis der Zahlung {{ paymentName }} hat sich während des Bestellvorgangs geändert. Überprüfen Sie bitte Ihre Bestellung."
 
 msgid "The price of shipping {{ transportName }} changed during ordering process. Check your order, please."
-msgstr "(de) The price of shipping {{ transportName }} changed during ordering process. Check your order, please."
+msgstr "Der Preis für den Versand {{ transportName }} hat sich während des Bestellvorgangs geändert. Überprüfen Sie bitte Ihre Bestellung."
 
 msgid "This account doesn't exist or password is incorrect"
-msgstr "(de) This account doesn't exist or password is incorrect"
+msgstr "Dieses Konto existiert nicht oder das Passwort ist falsch"
 
 msgid "This category contains no products."
-msgstr "(de) This category contains no products."
+msgstr "Diese Kategorie enthält keine Produkte."
 
 msgid "This product is no longer available for purchase. Have a look at similar products <a href=\"%url%\">here</a>."
-msgstr "(de) This product is no longer available for purchase. Have a look at similar products <a href=\"%url%\">here</a>."
+msgstr "Dieses Produkt kann nicht mehr gekauft werden. Werfen Sie <a href=\"%url%\">hier</a> einen Blick auf ähnliche Produkte."
 
 msgid "This site uses cookies."
-msgstr "(de) This site uses cookies."
+msgstr "Diese Website verwendet Cookies."
 
 msgid "This site uses cookies. By staying on the site you agree with that."
-msgstr "(de) This site uses cookies. By staying on the site you agree with that."
+msgstr "Diese Website verwendet Cookies. Indem Sie diese Webseite weiter nutzen, stimmen Sie der Cookie-Nutzung zu."
 
 msgid "To"
-msgstr "(de) To"
+msgstr "Zu"
 
 msgid "To prevent the possibility of account theft, the change of the e-mail address must be dealt with individually. If your e-mail address has changed, please contact us."
-msgstr "(de) To prevent the possibility of account theft, the change of the e-mail address must be dealt with individually. If your e-mail address has changed, please contact us."
+msgstr "Um die Möglichkeit eines Accountdiebstahls zu verhindern, muss die Änderung der E-Mail-Adresse individuell behandelt werden. Wenn sich Ihre E-Mail-Adresse geändert hat, kontaktieren Sie uns bitte."
 
```

Comment:
`TOP` should be translated as provided in https://github.com/shopsys/demoshop/pull/71#issuecomment-528836928 even if the translation is the same. It will prevent Symfony from warning about missing translation.

```suggestion

msgid "TOP"
msgstr "TOP"

```